app/views/jobs/_form.html.erb

```diff
@@ -1,15 +1,23 @@
 <%= form_for [@company, @job] do |f| %>
   <%= f.label :title %>
   <%= f.text_field :title %>
-
+  <br>
   <%= f.label :description %>
   <%= f.text_field :description %>
+  <br>
 
   <%= f.label :level_of_interest %>
-  <%= f.number_field :level_of_interest %>
+  <%= f.number_field :level_of_interest, min:1, max:150 %>
+  <br>
 
   <%= f.label :city %>
   <%= f.text_field :city %>
+  <br>
+
+  <%= f.label :category_id %>
+  <%= f.collection_select :category_id, Category.all, :id, :title %>
```

Comment:
this is also too much logic - what part of MVC should be responsible for grabbing our data and sending it to our view?